da.json

```diff
@@ -83,6 +83,11 @@
                 "trigger" : "Sprog",
                 "title" : "Vælg sprog",
                 "currentLanguage" : "Du bruger i øjeblikket [language]."
```

Comment:
A comma is missing at the end of this line. The file will break without it because the JSON is invalid.